### PR TITLE
HDS-1364 documentation scaffold script

### PR DESCRIPTION
**Pull request by mikkojamG**
## Description

Add scaffold script for documentation, similar what can be found in the `react` package.

## Related Issue

https://helsinkisolutionoffice.atlassian.net/browse/HDS-1364

## Motivation and Context

Creating new component documentation involves lot of copy-pasting for all necessary pages. Scaffold script takes new component name as an input and creates pages with some generic content accordingly.

## How Has This Been Tested?
- Tested on local machine

### CODE REVIEW

**Review by harriplappalainen — APPROVED** [review]
Excellent work, can't find anything to remark about🏅

**Review by minevala — COMMENTED** [review]
Works nicely 👍 
I had couple of suggestions

File: site/README.md [hunk 1]
```diff
@@ -49,6 +49,20 @@ yarn start
...
 **Note!** You may need to rerun `yarn build` if you make changes to config files or if other HDS packages are updated.
 
+### Commands
+
+| Command                            | Description                                               |
+| ---------------------------------- | --------------------------------------------------------- |
+| yarn                               | Install dependencies.                                     |
+| yarn build                         | Lints and builds documentation.                           |
+| yarn start                         | Starts the development server.                            |
+| yarn test                          | Runs the tests.                                           |
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply site/README.md]: Removed: https://github.com/City-of-Helsinki/helsinki-design-system/pull/953/commits/2f0c5e00329f54d53e6824b01bbd7d6188576c6f

File: site/.templates/new-component/code.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Code
+
+### Code examples
+
+### Packages
+
+### Properties
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply site/.templates/new-component/code.mdx]: Why not 🙂 Included: https://github.com/City-of-Helsinki/helsinki-design-system/pull/953/commits/2f0c5e00329f54d53e6824b01bbd7d6188576c6f

File: site/.templates/new-component/index.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+### Principles
+
+### Variations
```
[the remark this answers is not in the document]
Comment (mikkojamG) [reply site/.templates/new-component/index.mdx]: Included: https://github.com/City-of-Helsinki/helsinki-design-system/pull/953/commits/2f0c5e00329f54d53e6824b01bbd7d6188576c6f

**Review by minevala — APPROVED** [review]
Nice work 👍 
Helps to build component documentation in the future.

**Comment by harriplappalainen** [comment]
Ack, I forgot to merge this with the release 🤦 Might be ok to merge whenever though as it doesn't bring any value to users? Also note that there's a conflict in site/package.json.

**Pull request merged**